src/cliActions.js

```diff
@@ -153,17 +157,21 @@ const streamDApp = ({ pw }, signingKeyFile, devMode) =>
 const buildDApp = ({ pw }, signingKeyFile, devMode) =>
   new Promise((res, rej) => {
     checkExistAndDecryptSigningKey({ pw }, signingKeyFile)
-      .then(() => {
+      .then(singingPrivateKey => {
         // read signing key
         const rawSigningKey = fs.readFileSync(signingKeyFile, 'utf8');
         const signingKey = JSON.parse(rawSigningKey);
-        watchAndWriteBundleFile(devMode, signingKey, ({ error }) => {
-          if (!error) {
-            res('wrote dapp build to dapp_build.json');
-          } else {
-            rej(error);
-          }
-        });
+        watchAndWriteBundleFile(
+          devMode,
+          { ...signingKey, singingPrivateKey },
+          ({ error }) => {
+            if (!error) {
+              res('wrote dapp build to dapp_build.json');
+            } else {
```

Comment:
you don't need the `else`

Reply:
why not i could get error when try to write file

Reply:
```js
if (!error) {
   return res("bla")
}
rej("error")
```
it does the same with less code

Reply:
done